ipywidgets/widgets/widget.py

```diff
@@ -13,7 +13,7 @@
 from ipykernel.comm import Comm
 from traitlets.config import LoggingConfigurable
 from traitlets.utils.importstring import import_item
-from traitlets import Unicode, Dict, Instance, List, Int, Set, Bytes
+from traitlets import Unicode, Dict, Instance, List, Int, Set, Bytes, Bool
```

Comment:
Whoops, that was accidentally left over from when I used a widget implementation to check version.  It's removed now.